packages/compass-assistant/src/components/assistant-chat.tsx

```diff
@@ -203,6 +203,10 @@ export const AssistantChat: React.FunctionComponent<AssistantChatProps> = ({
 }) => {
   const track = useTelemetry();
   const darkMode = useDarkMode();
+  const messagesContainerRef = useRef<HTMLDivElement>(null);
+  const previousLastMessageId = useRef<string | undefined>(undefined);
+  const { id: lastMessageId, role: lastMessageIsUser } =
```

Comment:
Isn't role `"system" | "user" | "assistant"` yet here you're using it as a boolean lastMessageIsUser?